src/unit-stack-configuration-workspace/unit-stack-configuration.repository.spec.ts

```diff
@@ -79,4 +79,32 @@ describe('UnitStackConfigurationWorkspaceRepository', () => {
       expect(result).toEqual([unitStackConfigs]);
     });
   });
+
+  describe('getUnitStackConfigsByUnitId', () => {
+    it('calls createQueryBuilder and gets all Unit Stack Configations from the repository when it is a unit', async () => {
+      repository.createQueryBuilder = jest.fn().mockReturnValue(queryBuilder);
+      queryBuilder.innerJoinAndSelect.mockReturnValue(queryBuilder);
+      queryBuilder.innerJoinAndSelect.mockReturnValue(queryBuilder);
```

Comment:
I think you only need one mockReturnValue unless the thing returned is going to change